nodejs/eks/cni.ts

```diff
@@ -86,6 +86,24 @@ export interface VpcCniOptions {
      * Defaults to the official AWS CNI image in ECR.
      */
     image?: pulumi.Input<string>;
+
+    /**
+     * Specifies the veth prefix used to generate the host-side veth device
+     * name for the CNI.
+     *
+     * The prefix can be at most 4 characters long.
+     *
+     * Defaults to "eni".
+     */
+    vethPrefix?: pulumi.Input<string>;
+
+    /**
+     * Used to configure the MTU size for attached ENIs. The valid range is
+     * from 576 to 9001.
+     *
+     * Defaults to 9001.
+     */
+    eniMtu?: pulumi.Input<number>;
 }
```

Comment:
I don't see the following configs that are present in the upstream docs:
- ENI_CONFIG_ANNOTATION_DEF
- ENI_CONFIG_LABEL_DEF
- AWS_VPC_K8S_CNI_RANDOMIZESNAT
- AWS_VPC_K8S_CNI_EXCLUDE_SNAT_CIDRS
- MINIMUM_IP_TARGET
- MAX_ENI
- INTROSPECTION_BIND_ADDRESS
- DISABLE_INTROSPECTION
- DISABLE_METRICS
- ADDITIONAL_ENI_TAGS
- CLUSTER_NAME

Is that intentional?

Reply:
It's possible that these envvars did not exist at the initial time `cni.ts` was written. We've also have not gotten user demand to add them in.

Adding them in is straight forward, but I'm not sure if we want to include all envvars now and those forthcoming in the future, and I'm now reevaluating what the stance should be.

cc @lukehoban ^

Reply:
Opened an issue to track missing vars: https://github.com/pulumi/pulumi-eks/issues/337